tests/unit/htdocs/class/cache/XoopsCacheTest.php

```
@@ -10,6 +10,36 @@ class XoopsCacheTest extends TestCase
+    public static function tearDownAfterClass(): void
+    {
+        if (!is_dir(self::$cachePath)) {
+            return;
+        }
+        // Recursively remove all cache artifacts
+        $files = new \RecursiveIteratorIterator(
+            new \RecursiveDirectoryIterator(self::$cachePath, \FilesystemIterator::SKIP_DOTS),
+            \RecursiveIteratorIterator::CHILD_FIRST
+        );
+        foreach ($files as $file) {
+            $ok = $file->isDir()
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ok. Configured minimum length is 3. (undefined)

(ShortVariable)